src/main/java/com/timelogr/enterprise/TimeLogrController.java

```diff
@@ -5,51 +5,55 @@
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.RequestMapping;
 
-//ALL WEB FILES GO IN THE resources/templates DIRECTORY
+/**
+ * ALL WEB FILES GO IN THE resources/templates DIRECTORY
+ */
```

Comment:
This looks like a note to other developers.  As long as you're using JavaDoc, use JavaDoc to describe this class.